pkg/util/mount/nsenter_mount.go

```diff
@@ -188,13 +190,15 @@ func (n *NsenterMounter) IsLikelyNotMountPoint(file string) (bool, error) {
 		// It's safer to assume that it's not a mount point.
 		return true, nil
 	}
-	strOut := strings.TrimSuffix(string(out), "\n")
+	strOut := strings.Split(string(out), " ")[0]
```

Comment:
@jingxu97 Let's make the code a little easier to follow while you're in here.  Would you rename `strOut` to `mountTarget` ?